src/site/filters/liquid.js

```diff
@@ -710,4 +710,6 @@ module.exports = function registerFilters() {
   };
 
   liquid.filters.concat = (...args) => _.concat(...args);
+
+  liquid.filters.strip = (string = '') => _.trim(string);
```

Comment:
Can't believe there isn't a built-in filter for this lol.

Reply:
right? haha